src/mod/dns.mod/coredns.c

```diff
@@ -1196,6 +1196,42 @@ static void dns_lookup(sockname_t *addr)
   sendrequest(rp, T_PTR);
 }
 
+static int dns_hosts(char *hostn) {
+  FILE* f;
+  char line[1024];
```

Comment:
is getline() too GNU specific?

Reply:
No, this has nothing to do with GNU but with eggdrops POSIX 2001 compatibility. getline() is not posix 2001 but posix 2008. see http://www.unix.org/version3/inttables.pdf and/or https://www.freebsd.org/cgi/man.cgi?query=getline&apropos=0&sektion=0&manpath=FreeBSD+12.0-RELEASE+and+Ports&arch=default&format=html.